whylog_vim/tests/tests_proxy/tests_parser_menu_operations.py

```diff
@@ -1,10 +1,18 @@
-import six
 import platform
-from unittest import TestCase, skipIf
-from mock import patch, call
-from . import create_mock_editor
-from whylog_vim.proxy import WhylogProxy
+from unittest import TestCase
+
+import six
+from mock import call, patch
+
 from whylog_vim.consts import ReadMessages
+from whylog_vim.proxy import WhylogProxy
+from whylog_vim.tests.tests_proxy.utils import create_mock_editor
+
+try:
+    from unittest import skipIf
+except ImportError:
+    # for python 2.6
+    from whylog_vim.tests.tests_proxy.utils import skipIf
```

Comment:
Diverging testing framework across versions will make debugging harder.

Since we have to implement it for 2.6 anyway, just always import the decorator from `whylog_vim.tests.tests_proxy.utils` and just document in docstring that it is inspired by 2.7 skipIf from unittests.